_faq/application-template.md

```diff
@@ -1,8 +1,14 @@
 ---
 question: "Do you have an application template?"
 ---
-coala 2018 Application Template
+{% assign month = "now" | date: "%-m" | plus:0 %}
+{% if month >= 9 %}
```

Comment:
your solution is more complicated than the issue requested, but that is OK as the issue was a bit too easy for you, and you have correctly guessed that the requested solution was also not ideal.  We dumbed down the task a bit.

But, you created extra work for yourself ;-)

Now you have added a constant , and constants should go into

https://github.com/coala/projects/blob/master/_config.yml

`GSOC_SWITCH_MONTH: 7`

`lower_case` key names in that global file is also ok.

or maybe even June?
(September is a bit too late -- we get questions much earlier)